Makefile

```diff
@@ -2,10 +2,7 @@ TAG := $(shell git rev-parse --short HEAD)
 DIR := $(shell pwd -L)
 
 dep:
-	docker run -ti \
-        --mount src="$(DIR)",target="$(DIR)",type="bind" \
-        -w "$(DIR)" \
-        asecurityteam/sdcli:v1 go dep
+	GO111MODULE=on go mod vendor
```

Comment:
Is the command no longer running inside the container intentional? I.e. I'd expect this to  be
docker run -e GO111MODULE=on
vs direct invocation outside the container.

Reply:
Yeah, open source projects should build just fine in the container as they only rely on other open source projects. Fetching dependencies on the host is used as a work around for internal projects that require SSH keys for authentication.

Reply:
so sdlci doesn't have the `go mod vendor` command available to it
```
dep:
	docker run -ti \
        --mount src="$(DIR)",target="$(DIR)",type="bind" \
        -w "$(DIR)" \
        asecurityteam/sdcli:v1 go mod vendor
```
fails with
```
asecurityteam/sdcli:v1 go mod vendor
/usr/bin/sdcli: line 44: sdcli_go_mod_vendor: command not found
```

Reply:
updating to the latest image of sdcli seems ot have fixed it